portfolio/src/main/webapp/index.html

```diff
@@ -1,83 +1,105 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="UTF-8" />
-        <title>My Portfolio</title>
-        <link rel="stylesheet" href="node_modules/bulma/css/bulma.min.css" />
-        <link rel="stylesheet" href="style.css" />
-        <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
-        <script src="node_modules/jquery/dist/jquery.min.js"></script>
-    </head>
-    <body>
-        <section class="hero is-light is-large">
-            <div class="hero-head">
-                <nav class="navbar">
-                    <div class="container">
-                        <div class="navbar-brand">
-                            <a class="navbar-item">
-                                <h1>Iris Liu</h1>
-                            </a>
-                        </div>
-                        <div id="navbarMenuHeroB" class="navbar-menu">
-                            <div class="navbar-end">
-                                <a class="navbar-item is-active" href="index.html">Home</a>
-                                <a class="navbar-item" href="me.html">About Me</a>
-                                <a class="navbar-item" href="photo.html">Photography</a>
-                                <span class="navbar-item">
-                                    <a class="button is-inverted navbar-icon-background" href="https://linkedin.com/in/jiayi-liu-1010" target="_blank">
-                                        <span class="icon has-text-dark is-large">
-                                            <i class="fab fa-linkedin"></i>
-                                        </span>
-                                    </a>
-                                </span>
-                                <span class="navbar-item">
-                                    <a class="button is-inverted navbar-icon-background" href="https://github.com/irisliu77" target="_blank">
-                                        <span class="icon has-text-dark is-large">
-                                            <i class="fab fa-github-square"></i>
-                                        </span>
-                                    </a>
-                                </span>
-                            </div>
-                        </div>
-                    </div>
-                </nav>
-            </div>
+  <head>
```

Comment:
Can you reformat this? You lost the indentation fixes you made in #1

Reply:
i think i messed it up when dealing with merge conflict, should be right now